common/djangoapps/student/views.py

```diff
@@ -1541,7 +1541,7 @@ def user_signup_handler(sender, **kwargs):  # pylint: disable=unused-argument
             log.info(u'user {} originated from a white labeled "Microsite"'.format(kwargs['instance'].id))
 
 
-def _do_create_account(form, custom_form=None):
+def _do_create_account(form, custom_form=None, site=None):
```

Comment:
Currently, there are management commands that call this function directly to create user accounts. Adding a site option to these management commands is out of scope for this work.